controllers/dscinitialization/dscinitialization_controller.go

```diff
@@ -91,7 +91,7 @@ func (r *DSCInitializationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		instance = &instances.Items[0]
 	case len(instances.Items) > 1:
 		message := fmt.Sprintf("only one instance of DSCInitialization object is allowed. Update existing instance name %s", req.Name)
-		_, _ = r.updateStatus(ctx, instance, func(saved *dsciv1.DSCInitialization) {
+		_, _ = r.updateStatus(ctx, &instances.Items[0], func(saved *dsciv1.DSCInitialization) {
```

Comment:
Can you guarantee an order of dsci items? From the looks of it; when multiple dscis are in place; mark a random one as error.

A better option would probably be to get the DSCI Instance(Request) that triggered the Reconcile and assume that is the +1 and mark that as error.

Reply:
the "Creation time" is the correct one we should keep in the cluster, and the others are the ones we do not want.